Client/src/bkr/client/wizard.py

```diff
@@ -364,24 +364,24 @@
 WizardVersion = "2.3.2"
 
 # Regular expressions
-RegExpPackage    = re.compile("^(?![._+-])[.a-zA-Z0-9_+-]+(?<![._-])$")
-RegExpRhtsRequires = re.compile("^(?![._+-])[.a-zA-Z0-9_+-/()]+(?<![._-])$")
-RegExpPath       = re.compile("^(?![/-])[a-zA-Z0-9/_-]+(?<![/-])$")
-RegExpTestName   = re.compile("^(?!-)[a-zA-Z0-9-_]+(?<!-)$")
-RegExpReleases   = re.compile("^(?!-)[a-zA-Z0-9-_]+(?<!-)$")
-RegExpBug        = re.compile("^\d+$")
-RegExpBugLong    = re.compile("^bz\d+$")
-RegExpBugPrefix  = re.compile("^bz")
-RegExpCVE        = re.compile("^\d{4}-\d{4,}$")
-RegExpCVELong    = re.compile("^CVE-\d{4}-\d{4}$")
-RegExpCVEPrefix  = re.compile("^CVE-")
-RegExpEmail      = re.compile("^[a-z._-]+@[a-z.-]+$")
-RegExpYes        = re.compile("Everything OK|y|ye|jo|ju|ja|ano|da", re.I)
-RegExpReproducer = re.compile("repr|test|expl|poc|demo", re.I)
-RegExpScript     = re.compile("\.(sh|py|pl)$")
-RegExpMetadata   = re.compile("(\$\(METADATA\):\s+Makefile.*)$", re.S)
-RegExpTest       = re.compile("TEST=(\S+)", re.S)
-RegExpVersion    = re.compile("TESTVERSION=([\d.]+)", re.S)
+RegExpPackage    = re.compile(r"^(?![._+-])[.a-zA-Z0-9_+-]+(?<![._-])$")
+RegExpRhtsRequires = re.compile(r"^(?![._+-])[.a-zA-Z0-9_+-/()]+(?<![._-])$")
+RegExpPath       = re.compile(r"^(?![/-])[a-zA-Z0-9/_-]+(?<![/-])$")
+RegExpTestName   = re.compile(r"^(?!-)[a-zA-Z0-9-_]+(?<!-)$")
+RegExpReleases   = re.compile(r"^(?!-)[a-zA-Z0-9-_]+(?<!-)$")
```

Comment:
I don't think we need `r' here. I don't see any backslashes that could cause problems. But I guess it will not hurt anyway.

Reply:
I left them in for consistency jic somebody changes one them